### PR TITLE
Notification: Number Badge on AppBar

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1538
[Problem] Notification on AppBar is not implemented
[Solution]
 - added examples in UIComponents
 - Badge implementation according to guideline

[Screenshot]
obraz

obraz

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+					</option>
+				</select>
+			</div>
+		</header>
+
+		<div class="ui-content">
+		</div>
+
+		<script>
+			var dropDownMenu = document.getElementById("sample-dropdown-menu");
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html]: Why don't you move the content to a js file?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

File: examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html]: I'll fix it.

**Review by singa2000 — APPROVED** [review]
LGTM

**Pull request merged**